database-setup/initfile.sql

```diff
@@ -2,6 +2,20 @@ CREATE DATABASE IF NOT EXISTS llanwrytd;
 
 USE llanwrytd;
 
+CREATE TABLE userData (
+    userID VARCHAR(128) NOT NULL,
+    userEmail VARCHAR(128),
+    userPassword VARCHAR(256),
+    userRole ENUM('admin', 'non-admin', 'pending'),
+    PRIMARY KEY ( userID )
```

Comment:
Could we use an auto-incrementing `INT` for the primary key? Has a few benefits, the least of which being easier range selection, and a more distinguishable ID (cf: USER, US3R).

Reply:
Sure, I can add a column named `userIndex` and make it the primary key.

Reply:
Sounds good, feel free to merge afterwards 👍

Reply:
issue resolved in the commit [4b12c77](https://github.com/aimlac-7564616D/composer/pull/9/commits/4b12c774926b6147fc12359b1b68d2aadd97c1d2)